simulator-core/src/test/java/com/evbox/everon/ocpp/functional/availability/ChangeAvailabilityChargingStationTest.java

```diff
@@ -1,37 +1,31 @@
-package com.evbox.everon.ocpp.simulator.it;
+package com.evbox.everon.ocpp.functional.availability;
```

Comment:
suggesting to rename  _functional_ to **it** package

Reply:
done